src/Databricks/Az.Databricks.psd1

```diff
@@ -12,7 +12,7 @@
 RootModule = './Az.Databricks.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.1.1'
+ModuleVersion = '0.2.0'
```

Comment:
version should not be changed. Please check with Yunchi. He knows the issue I met.

Reply:
Talked with Yunchi and I rolled it back. Thanks